cmd/ko/README.md

```diff
@@ -166,6 +166,54 @@ to whatever `kubectl` context is active.
 commands. It is exposed purely out of convenience for cleaning up resources
 created through `ko apply`.
 
+## Configuration via `.ko.yaml`
+
+While `ko` aims to have zero configuration, there are certain scenarios where
+you will want to override `ko`'s default behavior. This is done via `.ko.yaml`.
+
+`.ko.yaml` is put into the directory from which `ko` will be invoked.  If it
+is not present, then `ko` will rely on its default behaviors.
+
+### Overriding the default base image
+
+By default, `ko` makes use of `gcr.io/distroless/base:latest` as the base image
+for containers.  There are a wide array of scenarios in which overriding this
+makes sense, to name a few:
+1. Pinning to a particular digest of this image for repeatable builds,
+1. Replacing this streamlined base image with another with better debugging
```

Comment:
Maybe link to https://github.com/GoogleContainerTools/distroless#debug-images

Reply:
I was thinking ubuntu